src/calculateRentalCost.js

```diff
@@ -27,20 +27,26 @@
  * @return {number}
  */
 
+// my pull request
+
 function calculateRentalCost(days) {
   // write code here
   const rentCost = 40;
   const totalRentCost = rentCost * days;
+  const bigRentDiscount = 50;
+  const smallRentDiscount = 20;
+  const firstDaysCount = 3;
+  const secondDaysCount = 7;
```

Comment:
```suggestion
  const smallDiscountPeriod = 3;
  const bigDiscountPeriod = 7;
```
first и second не особо понятно почему, какой из них первый, а какой второй?
Какой большой, а какой маленький понятно сразу